src/main/java/DukeList.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+
+public class DukeList {
+    private ArrayList<Task> list;
+
+    public DukeList() {
+        this.list = new ArrayList<Task>();
+    }
+
+    public void add(Task item) {
+        list.add(item);
+    }
+
+    public void done(int x) {
```

Comment:
Since this is a function that does an action, would it be better to name it in a way that it sounds like a function?